### PR TITLE
css files and background

### PR DESCRIPTION
**Pull request by Birek100**
Add file style.css and background file

### CODE REVIEW

File: public/styles.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+body {
+   background: url("background.jpg");
```
Comment (oskarszura) [inline public/styles.css]: I don't see `background.jpg` added to the repository, where is it?

**Comment by oskarszura** [comment]
Typo in PR description ;) 
is:
> Add file style.css an background file
should be:
> Add file style.css and background file

File: public/styles.css (hunk 1 above)
Comment (Birek100) [reply public/styles.css]: fixed

**Comment by Birek100** [comment]
fixed

**Comment by oskarszura** [comment]
👍 , please merge it!

**Pull request merged**